docs/release_notes/2024/august2024.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+id: August_2024
+description: Release notes for August
+title: August 2024
+tags:
+  - release notes
+  - changelog
+  - august
+---
+
+## 3.3.9.\* (August 2, 2024)
+
+- Prophecy Python libs version: 1.9.9
+- Prophecy Scala libs version: 8.0.29
+
+### Features {#Features339}
+
+#### Spark Copilot Enhancements
+
+- **Snowflake Partner Network Integration**: We've added a Connection String Identifier for Snowflake, following the Snowflake Native Connector Best Practices Guide. This was a requirement to advance in the Snowflake Partner Network.
+
+- **Support custom artifactory resolve plibs version**: We now support custom artifactory resolution mode on EMR Fabrics such as Livy.
+
+- **Object case sensitivity**: Table checks and lookups performed by Prophecy are no longer case sensitive.
+
+- **Support for config resolution inside call_func()**: Config variables inside of the `call_function()` and `call_func()` are now supported.
+
+- **Skip resubmission of Initial.Scala for Livy Fabrics**: In incremental mode, the Initial.Scala code would get submitted for every play button. We've introduced caching to avoid resubmission of the same code.
```

Comment:
This is not fully done feature. Lets remove for now.

Reply:
Done.